packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -3306,6 +3306,37 @@ describe('TableView', function () {
 
         scrollHeight.mockRestore();
       });
+
+      // To test https://github.com/adobe/react-spectrum/issues/1885
+      it('should not throw error if selection mode changes with overflowMode="wrap" and selection was controlled', function () {
```

Comment:
i know you probably ran this test before you made the change, but given that the description is fulfilled implicitly by making it all the way through this test and there is no explicit check for a not to throw at any point, can you confirm?
maybe the thing to do is add a comment at the point where it was throwing just so that if anyone breaks it in that way, they'll know if it was the same issue vs something new

Reply:
sure, I'll add a comment. For posterity, the error thrown happens at the rerender and looks like this:
```
 Uncaught [TypeError: Cannot set property 'estimatedSize' of undefined]
```